src/main/java/io/lighty/yang/validator/formats/yang/printer/Util.java

```diff
@@ -22,9 +22,5 @@ private Util() {
         }
         return typeDef;
     }
-
```

Comment:
Why removed?

Reply:
Unused protected method. Done in cleanup.